pkg/test-infra/fixture/fixture.go

```diff
@@ -221,9 +221,9 @@ func init() {
 	flag.StringVar(&Context.LokiPassword, "loki-password", "", "loki password. Needed when basic auth is configured in loki")
 
 	flag.StringVar(&Context.TiDBClusterConfig.ImageVersion, "image-version", "nightly", "image version")
-	flag.StringVar(&Context.TiDBClusterConfig.TiDBImageVersion, "tidb-image", "", "tidb image version")
-	flag.StringVar(&Context.TiDBClusterConfig.TiKVImageVersion, "tikv-image", "", "tikv image version")
-	flag.StringVar(&Context.TiDBClusterConfig.PDImageVersion, "pd-image", "", "pd image version")
+	flag.StringVar(&Context.TiDBClusterConfig.TiDBImage, "tidb-image", "", "tidb image version")
```

Comment:
```suggestion
	flag.StringVar(&Context.TiDBClusterConfig.TiDBImage, "tidb-image", "", "tidb image")
```

And could you change `TiFlashImageVersion` in the same way?